src/chapters/radius-auth.xml

```diff
@@ -0,0 +1,251 @@
+            <varlistentry>
+                <term><property>radius-key-file</property></term>
+                <listitem>
+                    <para>The combination certificate and private key pair to use for TLS-based
+                        RADIUS protocols that require a client-side certificate.  This parameter
+                        should specify the absolute path to the file.  By default the extension
+                        will look for a file called radius.key in the GUACAMOLE_HOME directory.
+                        </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                 <term><property>radius-key-type</property></term>
+                 <listitem>
+                     <para>The type of the key file specified by the <property>radius-key-file</property> 
+                        parameter.  If not specified, this defaults to pkcs12, the default used by 
+                        the JRadius library.</para>
+                 </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-key-password</property></term>
+                <listitem>
+                    <para>The password of the private key specified in the 
+                        <property>radius-key-file</property> parameter.  By default the extension   
+                        will not use any password when trying to open the key file.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-ca-file</property></term>
+                <listitem>
+                    <para>The absolute path to the file that stores the certificate authority
+                        certificates for encrypted connections to the RADIUS server.  By default
+                        a file with the name ca.crt in the GUACAMOLE_HOME directory will be used.
+                        </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-ca-type</property></term>
+                <listitem>
+                    <para>The type of file store used for the certificate authority.  If not
+                        specified this defaults to pem.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-ca-password</property></term>
+                <listitem>
+                    <para>The password used to protect the certificate authority store, if
+                        any.  If unspecified the extension will attempt to read the CA
+                        store without any password..</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-trust-all</property></term>
+                <listitem>
+                    <para>A boolean value indicating whether or not the RADIUS extension
```

Comment:
Similar to the problem with telling users the default is "null", documenting that the property is a boolean value is not necessarily sufficient to inform users that the property can be set to "true" or "false". The default is nicely documented here, but the fact that this should be set to "true" to trust all certificates needs to be explicit.

Reply:
Fixed.